solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -486,23 +491,20 @@ contract RandomBeacon is Ownable {
             "Operator is already registered"
         );
 
-        sortitionPool.insertOperator(operator);
+        sortitionPool.insertOperator(
+            operator,
+            staking.eligibleStake(msg.sender, address(this))
```

Comment:
in line 487 we define `operator = msg.sender`. Should we use `operator` here for readibility as well? Or remove operator and use msg.sender instead

Reply:
Good point, done in https://github.com/keep-network/keep-core/pull/2737/commits/d42244fa2b145b5d2d07a154c6a0970233b71ca4.